pallets/staking-rewards/src/mock.rs

```diff
@@ -607,11 +706,29 @@ pub(crate) fn run_for_n_blocks(n: u64) {
     }
 }
 
+pub fn roll_to_unsigned() {
```

Comment:
mb keep naming run_to?

Reply:
This is the original naming from the multi-phase election pallet tests in Substrate. Sure it'd be better to align it with what we have.